source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,69 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.unminimized-comment [aria-label="Edit comment"]:not([hidden])'],
+	['d', '[aria-label="Delete comment"]']
+]);
+
+function runShortcuts(event: KeyboardEvent): void {
+	const focusedComment = select(':target');
+	if (isEditable(event.target) || !focusedComment) {
+		return;
+	}
```

Comment:
If you exit here, then `currentIndex` will never be `-1`

```suggestion
	if (isEditable(event.target)) {
		return;
	}
	
	const focusedComment = select(':target');
```

If a comment isn't selected, j/k will select one.